conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,206 @@
+Committee suggests mediation, it should:
+
+* Find a candidate who can serve as a mediator.
+* Obtain the agreement of the reporter(s). The reporter(s) have complete
+  freedom to decline the mediation idea, or to propose an alternate mediator.
+* Obtain the agreement of the reported person(s).
+* Settle on the mediator: while parties can propose a different mediator than
+  the suggested candidate, only if common agreement is reached on all terms can
+  the process move forward.
+* Establish a timeline for mediation to complete, ideally within two weeks.
+
+The mediator will engage with all the parties and seek a resolution that is
+satisfactory to all.  Upon completion, the mediator will provide a report
+(vetted by all parties to the process) to the Committee, with recommendations
+on further steps.  The Committee will then evaluate these results (whether
+satisfactory resolution was achieved or not) and decide on any additional
+action deemed necessary.
+
+
+## How the committee will respond to reports
+
+When a report is sent to the committee they will immediately reply to the
+reporter to confirm receipt. This reply must be sent within 24 hours, and the
+group should strive to respond much quicker than that.
+
+See the [Reporting Guidelines](*conduct_reporting_online.md*) for details of
+what reports should contain. If a report doesn't contain enough information, the
+committee will obtain all relevant data before acting. The committee is
+empowered to act on the Steering Council’s behalf in contacting any individuals
+involved to get a more complete account of events.
+
+The committee will then review the incident and determine, to the best of their
+ability:
+
+* What happened.
+* Whether this event constitutes a Code of Conduct violation.
+* Who are the responsible party(ies).
+* Whether this is an ongoing situation, and there is a threat to anyone's
+  physical safety.
+
+This information will be collected in writing, and whenever possible the
+group's deliberations will be recorded and retained (i.e. chat transcripts,
+email discussions, recorded conference calls, summaries of voice conversations,
+etc).
+
+It is important to retain an archive of all activities of this committee to
+ensure consistency in behavior and provide institutional memory for the
+project.  To assist in this, the default channel of discussion for this
+committee will be a private mailing list accessible to current and future
+members of the committee as well as members of the Steering Council upon
+justified request. If the Committee finds the need to use off-list communications (e.g. phone calls for early/rapid response), it should in all cases summarize these back to the list so there's a good record of the process.
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within
+one week. In the event that a resolution can't be determined in that time, the
+committee will respond to the reporter(s) with an update and projected timeline
```

Comment:
The best I can think to do is a sentence to clarify that an anonymous submission has significant inherent limitations, and that we suggest instead people create at least a throw-away email address without their real name.  

This is basically as good as anonymous, but gives us a chance to communicate back.  Otherwise with a true one-off, 100% anonymous report, there will be limits to what we can realistically do.